src/input-helper.ts

```diff
@@ -0,0 +1,104 @@
+import * as core from '@actions/core'
+import * as fsHelper from './fs-helper'
+import * as github from '@actions/github'
+import * as path from 'path'
+import {ISourceSettings} from './git-source-provider'
+
+export function getInputs(): ISourceSettings {
+  const result = ({} as unknown) as ISourceSettings
```

Comment:
will `{} as ISouceSettings` work?

Reply:
unfortunately no either the linting or style rules we have doesnt like it